### PR TITLE
add idea in .gitignore file.

### PR DESCRIPTION
**Pull request by akin11235**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -9,32 +9,33 @@ media
...
-.idea/**/sqlDataSources.xml
-.idea/**/dynamic.xml
-.idea/**/uiDesigner.xml
-.idea/**/dbnavigator.xml
-
-# Gradle
-.idea/**/gradle.xml
-.idea/**/libraries
+.idea
+# # If you are using PyCharm #
```
Comment (pkrzyz) [inline .gitignore]: Can we delete all the commented lines?

Comment (akin11235) [reply .gitignore]: Sure, I'll delete!

Comment (akin11235) [reply .gitignore]: All good, done!

**Review by pkrzyz — APPROVED** [review]
Great work!!

**Pull request merged**